docs/advanced/tls_hardware_acceleration.md

```diff
@@ -18,36 +18,38 @@ You may want to add hardware acceleration in the following cases:
 
 - Your platform has a dedicated crypto-module capable of executing cryptographic primitives, and possibly storing keys securely.
 
-The mbed TLS library was written in C and it has a small amount of hand-optimised assembly code, limited to arbitrary precision multiplication on some processors. You can find the list of supported platforms in the top comment in [bn_mul.h](https://github.com/ARMmbed/mbedtls/blob/development/include/mbedtls/bn_mul.h).
+The mbed TLS library was written in C and it has a small amount of hand-optimised assembly code, limited to arbitrary precision multiplication on some processors. You can find the list of supported platforms in the top comment in [`bn_mul.h`](https://github.com/ARMmbed/mbedtls/blob/development/include/mbedtls/bn_mul.h).
```

Comment:
- Please add a comma after "C" because two independent clauses joined by a coordinating conjunction require a comma before the conjunction.
- Please change "hand-optimised" to "hand-optimized" for American English.